cmd_login.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gustash/freecarnival/auth"
+	"github.com/gustash/freecarnival/logger"
+	"github.com/spf13/cobra"
+)
+
+func newLoginCmd() *cobra.Command {
+	var email string
+	var password string
+
+	cmd := &cobra.Command{
+		Use:   "login",
+		Short: "Log in to your IndieGala account",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if email == "" || password == "" {
+				return fmt.Errorf("--email and --password are required")
+			}
+
+			client, res, err := auth.Login(cmd.Context(), email, password)
+			_ = client // will be reused later
```

Comment:
This is actually not used, but I assume we can use this to sync the library immediately after login, which we should do